implementations/rust/ockam/ockam_command/src/enroll/command.rs

```diff
@@ -190,6 +159,62 @@ impl EnrollCommand {
 
         Ok(())
     }
+
+    async fn enroll_identity(
+        &self,
+        ctx: &Context,
+        opts: &CommandGlobalOpts,
+        node: &InMemoryNode,
+    ) -> miette::Result<UserInfo> {
+        // Force enrollment if there are no spaces or projects
+        let force_enrollment = opts.state.get_spaces().await?.is_empty()
```

Comment:
This piece of logic, for deciding if enrollment should happen or not, could go into the `CliState` (with a parameter to force the enrollment or not).

Reply:
Done!